locales/en/email-strings.ftl

```diff
@@ -88,3 +88,73 @@ share-by-email-message =
     1. Go to {"https://monitor.firefox.com"} and search your email.
     2. See if your online accounts have been exposed in a data breach.
     3. Get tips from {-product-name} about what to do next.
+
+
+# Unsubscribe link in email.
+email-unsub-link =  Unsubscribe
+
+email-footer-blurb = You’re receiving this email because you signed up for {-product-name} 
+  alerts. No longer want these emails? { $unsubLink }. This is an automated email. For support, visit { $faqLink }.
```

Comment:
Can you add a comment to explain what's the text used to replace the two variables?